.github/workflows/cd.yml

```diff
@@ -5,31 +5,101 @@ on: push
 env:
   CI: true
 jobs:
+  prepare-deployment:
+    runs-on: ubuntu-18.04
```

Comment:
I assume everything we do from now on should be on Ubuntu 20.04...?

Reply:
It's still in preview, but yeah, it'll likely work fine and most contributors and servers will be on that version, so we can update it as we go. I'll change it here.

![image](https://user-images.githubusercontent.com/4251/85264653-48912f80-b471-11ea-8c47-b69614d95a83.png)

Reply:
Fixed in 3cfeb03728a8c5a6b251c97c2fbc4df26f7f5644.